src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -24,6 +26,7 @@ def heartbeat():
 def create_app():
     app = Flask(__name__)
     app.add_url_rule("/heartbeat", view_func=heartbeat)
+    app.add_url_rule("/v1/traces", view_func=trace_collector, methods=["POST"])
```

Comment:
add the ui entry in next PR? 
/ui/traces?session=abc

Reply:
added this